tags/guide/mixinexport.ytag

```diff
@@ -0,0 +1,25 @@
+type: embed
+
+colour: black
+embed:
+    title: Exporting Mixin Classes
+    image:
+        url: "https://cdn.discordapp.com/attachments/523633816078647296/1176733253877641306/mixin-export.png"
```

Comment:
Can you move this image to the repo: https://github.com/FabricMC/community/tree/main/images

Reply:
Sure! Although there are several other tags with Discord CDN images. eg. cheatsheet/sides.ytag